Lesson2Xcode/SettingsViewController.swift

```diff
@@ -0,0 +1,95 @@
+//
+//  SettingsViewController.swift
+//  Lesson2Xcode
+//
+//  Created by Илья on 21.09.2021.
+//
+
+import UIKit
+
+class SettingsViewController: UIViewController, UIPickerViewDelegate, UIPickerViewDataSource {
+
+    //Outlets
+    @IBOutlet weak var settings_label: UILabel!
```

Comment:
```suggestion
    @IBOutlet weak var settingsLabel: UILabel!
```
в swift используется camelCase